public/pages/CreateMonitor/components/ClusterMetricsMonitor/utils/clusterMetricsMonitorConstants.js

```diff
@@ -104,6 +104,25 @@ export const API_TYPES = {
       source: 'ctx.results[0].nodes.NODE_ID.jvm.mem.heap_used_percent > 60',
     },
   },
+  CAT_INDICES: {
+    type: 'CAT_INDICES',
+    documentation: 'https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-indices/',
+    exampleText: 'index1,index2...',
+    label: 'Indices',
```

Comment:
Looking at what label we are using for the cat actions
cat pending tasks: List pending tasks
cat recovery: Recovery

And now Indices, Shards, not sure if these are consistent, maybe just change all to "List ...s"

Reply:
Thank you! I meant to go back and change that from the place holder text. I've updated `cat/indices` and `cat/shards` to `List ...`.